frontend/src/services/notebookService.ts

```diff
@@ -169,23 +191,54 @@ export const createNotebook = (
     });
 };
 
-export const deleteNotebook = (projectName: string, notebookName: string): Promise<Notebook> => {
-  const url = `/api/notebooks/${projectName}/${notebookName}`;
+/** We have a notebook, start it back up */
+const enableNotebook = async (notebook: Notebook): Promise<Notebook> => {
+  const { name, namespace } = notebook.metadata;
 
-  return axios
-    .delete(url)
-    .then((response) => {
-      return response.data;
-    })
-    .catch((e) => {
-      throw new Error(e.response.data.message);
-    });
+  if (!namespace) {
+    return Promise.reject('Notebook is not assigned to a namespace -- cannot start it');
+  }
+
+  return patchNotebook(
```

Comment:
Arguably `update` and not a `patch`... but it is what it is. Our server side logic will allow this to happen. When we bring the SDK into it, this will be converted to an update I think. Just easier to re-use existing logic.

If reviewers feel differently, I can change this to an update and implement the backend API for it.